contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -906,6 +906,31 @@ class tsqlCommonMutator : public TSqlParserBaseListener
 	void exitFunction_call(TSqlParser::Function_callContext *ctx) override
 	{
 		handleGeospatialFunctionsInFunctionCall(ctx);
+
+		if (ctx->func_proc_name_server_database_schema())
+		{
+			if (ctx->func_proc_name_server_database_schema()->procedure)
+			{
+				std::string proc_name = stripQuoteFromId(ctx->func_proc_name_server_database_schema()->procedure);
+
+				if (pg_strcasecmp(proc_name.c_str(), "trim") == 0 
+					&& !(ctx->func_proc_name_server_database_schema()->server)
+					&& !(ctx->func_proc_name_server_database_schema()->database)
+					&& !(ctx->func_proc_name_server_database_schema()->schema))
+				{
+					rewritten_query_fragment.emplace(std::make_pair(ctx->func_proc_name_server_database_schema()->start->getStartIndex(), std::make_pair(::getFullText(ctx->func_proc_name_server_database_schema()), "sys.trim")));
```

Comment:
Instead to replacing `trim -> sys.trim`, it's better to just replace `"" -> "sys."`, applies to other places as well.
```suggestion
					rewritten_query_fragment.emplace(std::make_pair(id->keyword()->TRIM()->getSymbol()->getStartIndex(), std::make_pair("", "sys.")));
```